docs/configuration/parsing-methods.rst

```diff
@@ -0,0 +1,57 @@
+.. _parsing-methods:
+
+Parsing Methods
+===============
+
+Cosmos offers several options to parse your dbt project:
+
+- ``automatic``. Tries to find a user-supplied ``manifest.json`` file. If it can't find one, it will run ``dbt ls`` to generate one. If that fails, it will use Cosmos' dbt parser.
+- ``dbt_manifest``. Parses a user-supplied ``manifest.json`` file. This can be generated manually with dbt commands or via a CI/CD process.
+- ``dbt_ls``. Parses a dbt project directory using the ``dbt ls`` command.
+- ``custom``. Uses Cosmos' custom dbt parser, which extracts dependencies from your dbt's model code.
+
+
+``automatic``
+-------------
+
+When you don't supply an argument to the ``load_mode`` parameter (or you supply the value ``"automatic"``), Cosmos will attempt the other methods in order:
+
+1. Use a pre-existing ``manifest.json`` file (``dbt_manifest``)
+2. Try to generate a ``manifest.json`` file from your dbt project (``dbt_ls``)
+3. Use Cosmos' dbt parser (``custom``)
+
+``dbt_manifest``
+----------------
+
+If you already have a ``manifest.json`` file created by dbt, Cosmos will parse the manifest to generate your DAG.
+
+You can supply a ``manifest_path`` parameter on the DbtDag / DbtTaskGroup with a path to a ``manifest.json`` file. For example:
+
+.. code-block:: python
+
+    DbtDag(
+        manifest_path="/path/to/manifest.json"
+        ...,
+    )
+
+``dbt_ls``
```

Comment:
I believe the key advantage of using this parsing mode is the fact that it filters using `dbt` itself, as opposed to our custom filtering logic. I believe it would be worth to explain the filtering strategy used for each parsing method, WDYT?